docker/generate_build_matrix.py

```diff
@@ -21,7 +21,7 @@
 
 def _get_pytorch_version(python_version: str):
     if python_version == '3.9':
-        return '1.11.0'
+        return '1.12.0'
     if python_version in '3.8':
         return '1.10.2'
     if python_version == '3.7':
```

Comment:
See the comment re the build matrix -- can we bump python 3.7 to torch 1.10, and python 3.8 to torch 1.11?